### DISCUSSION TITLE
[PrimeNg 16] TabMenu doesn't update when model changes

### DISCUSSION THREAD

**Discussion by GiangNguyenDev** [primefaces/community#74] [PrimeNG] [answered]
Hi,

I updated PrimeNg to version 16 and realise that when I change the model binding of my TabMenu, the UI doesn't update. Here is an example:

https://stackblitz.com/edit/hitagv?file=src%2Fapp%2Fdemo%2Ftab-menu-basic-demo.html

Can you check it please? Thanks.

**Comment by cetincakiroglu** [comment]
Hi,

We've fixed this [issue](https://github.com/primefaces/primeng/issues/13471) in 16.2.0 release. Could you please try it with the >= 16.2.0 versions

**Reply by GiangNguyenDev** [reply]
Thank you for your response. I'm using version 16.3.1 and the issue still exists.

**Reply by cetincakiroglu** [reply, marked as answer]
This issue is fixed, here I prepared a StackBlitz example by using your code in v16.3.1 https://stackblitz.com/edit/ynjca9?file=src%2Fapp%2Fdemo%2Ftab-menu-active-demo.ts,src%2Fapp%2Fdemo%2Ftab-menu-active-demo.html

**Reply by GiangNguyenDev** [reply]
Thank you, the show/hide function works now when model changes. But updating of label seems to have issue, but it is not part of this discussion. I will test more and create new thread if needed. Thanks again 🙂.
